tests/chocolatey-tests/commands/choco-upgrade.Tests.ps1

```diff
@@ -120,6 +120,33 @@ Describe "choco upgrade" -Tag Chocolatey, UpgradeCommand {
         }
     }
 
+    Context "Upgrade package with (<Command>) specified" -ForEach @(
+        @{ Command = '--pin' ; Contains = $true }
+        @{ Command = '' ; Contains = $false }
+    ) {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot -SetWorkDir

```

Comment:
You don't use anything in the working directory, because of this, you don't need to specify the `-SetWorkDir`.

It is only useful if you need the current `CWD` to be in the working directory.
Typically it is never needed.

Not requiring a change, just mentioning it so you are aware of it in the future.

Reply:
oooh, I think I copied the context block from somewhere else that used it. I've removed this from both the install and upgrade files.